### ISSUE TITLE
over-fitting 

### ISSUE THREAD

**Issue by WillLiGitHub** [Tencent/tencent-ml-images#68]
when you train multi-labels classification on you dataset, 
whether there is serious overfitting？
whether you novel loss function can resolve this problem?
